src/gurt/telemetry.c

```diff
@@ -254,15 +258,37 @@ attach_shmem(key_t key, size_t size, int flags, struct d_tm_shmem_hdr **shmem)
 }
 
 static int
-new_shmem(key_t key, size_t size, struct d_tm_shmem_hdr **shmem)
+alloc_nsmem(size_t size, struct d_tm_shmem_hdr **shmem)
```

Comment:
tbh I'd rather have either `alloc_mem` or `alloc_non_shared_mem`. I can tell what `nsmem` means in the context of the PR, but outside of this diff, I think it's nonobvious.